### ISSUE TITLE
mAP on 12bit is only 0.3269

### ISSUE THREAD

**Issue by chenshen03** [Vijetha1/WDHT#2]
I used all the pre-training models and data you provided to test. But on 12 bits, the mAP is only 0.3269. Why is the result so different from that of the paper?

**Comment by chenshen03** [comment]
![image](https://user-images.githubusercontent.com/20432947/56486725-3c2ae300-650b-11e9-8667-e88a6286229b.png)

**Comment by Vijetha1** [comment]
We will look into this and get back to you. Will be busy this whole week. Please expect a delayed response.

**Comment by chenshen03** [comment]
@Vijetha1 Thank you for your help. I've been waiting for your reply. Have you found the reason for the problem?

**Comment by Vijetha1** [comment]
Hi Chen, trying to replicate this issue. Will update you asap.

**Comment by Vijetha1** [comment]
Hi Chen, I was not able to replicate the issue nor the set-up due to the lack of support for Theano. Instead, I recreated the code with Tensorflow backend and it is working as expected. Please verify now.

**Comment by chenshen03** [comment]
Thank you very much for your patient answer. I'll try again.

**Comment by Vijetha1** [comment]
Hi, were you able to verify this?

**Comment by ximi2018** [comment]
HI ，I cannot find the dataset and pretrain weight ，can you help me?

**Comment by mengliu1991** [comment]
@ximi2018 Can you download the dataset and weight? I cannot download them. Could you tell me how to deal with it?

**Comment by ximi2018** [comment]
@mengliu1991   sorry,I cannot get them

**Comment by chenshen03** [comment]
@mengliu1991 @ximi2018 Maybe you need a VPN. The link provided by the author is still available for download.

**Comment by mengliu1991** [comment]
@chenshen03 
Hi,
Have you test the L3 loss+L4 loss case, i.e., using the similarity matrix to train the model?
I change the model into the binary tag-vector model, but I cannot obtain the result reported in the paper.
